### PR TITLE
linux-kernel-lts: update to 6.6.54

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Topic Description
-----------------

- linux-kernel-lts: bump rel
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- Revert "linux-kernel-lts: build non-DRM simple framebuffer driver into 64-bit LoongArch kernel"
    This reverts commit 125ae2ca4bb20d6b4416e9779b33672c4368cfed.
- linux-kernel-lts: build non-DRM simple framebuffer driver into 64-bit LoongArch kernel
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- Revert "linux-kernel: build non-DRM simple framebuffer driver into 64-bit LoongArch kernel"
    This reverts commit f9d020a16388168280d5abf2fd1ca4de019134c7.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel: build non-DRM simple framebuffer driver into 64-bit LoongArch kernel
    Reported-by: 安慕希
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- Revert "linux-kernel-lts: build simple framebuffer driver into 64-bit LoongArch kernel"
    This reverts commit 69bb0883d1818d5e3d20dbe3228be5b699d8a838.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel-lts: build simple framebuffer driver into 64-bit LoongArch kernel
    Reported-by: 李克思
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel-lts: update to 6.6.54
    - Track patches at https://github.com/AOSC-Tracking/linux @ aosc/v6.6.54, current HEAD is d32192ba2151405f54443e0e547ad874115e6879.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel-lts: update Power ISA config for 6.6.54-rc2
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux+kernel+lts: update to 6.6.54
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>

... and 1 more commits

Package(s) Affected
-------------------

- linux+kernel+lts: 2:6.6.54
- linux-kernel-lts-${__VER}: 6.6.54-4

Security Update?
----------------

No

Build Order
-----------

```
#buildit linux-kernel-lts linux+kernel+lts
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by KexyBiscuit** [comment]
Smoke testing passed on AMD64 generation 2, AMD64 generation 1 and Armv8-A generation 2 Hyper-V VMs.

image
image
image

**Comment by KexyBiscuit** [comment]
Smoke testing passed on AMD64 VMware VM.

- Product: VMware® Workstation 17 Pro
- Version: 17.6.0 build-24238078
- Host OS version: Windows 11 Enterprise, 64-bit (Build 27718.1000) 10.0.27718
- Accelerate 3D graphics: on
- Firmware type: UEFI
- Host VBS Mode: on

image

**Comment by KexyBiscuit** [comment]
Smoke testing passed on Latitude D630.

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on 战神Z8-KP7S2.

图片
图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on RedmiBook 16 (锐龙版).

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on Surface Pro (5th Gen).

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on ROG Ally (2023).

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on ProArt B550-CREATOR.

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on AMD64 VMware VM with 6.6.54.

File: runtime-kernel/linux-kernel-lts/autobuild/ppc64el/config
Comment (MingcongBai) [inline runtime-kernel/linux-kernel-lts/autobuild/ppc64el/config:1]: Ditto from mainline, is this intended?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by MingcongBai** [comment]
Tested good on LoongArch (THTF Chaorui L680-T2).

File: runtime-kernel/linux-kernel-lts/autobuild/ppc64el/config
Comment (KexyBiscuit) [reply runtime-kernel/linux-kernel-lts/autobuild/ppc64el/config:1]: In https://github.com/torvalds/linux/commit/44ac4625ea002deecd0c227336c95b724206c698, the config is marked as broken.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**